lib/client.js

```diff
@@ -128,7 +128,13 @@ Client.prototype.request = function (method, path, data) {
             }
             resolve(res);
         });
-    }).bind(this);
+    }).bind(this)
```

Comment:
We should make this more transparent. 
```
const Beam = require('beam-client-node');
Beam.Promise = <Promise constructor you want>
```
Then `Beam.Promise` in place of Bluebird.

Or something.

Right now that's super unclear to me. As a short term i'll test this with the frontend promise provider.